ethereum/executionlayer/src/test/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionLayerManagerImplTest.java

```diff
@@ -842,7 +843,8 @@ private void verifyFallbackToLocalEL(
                             executionPayloadContext,
                             Optional.empty(),
                             Optional.empty(),
-                            Optional.of(SafeFuture.completedFuture(headerWithFallbackData))))))
+                            Optional.of(SafeFuture.completedFuture(headerWithFallbackData)),
+                            Optional.empty()))))
```

Comment:
I think here we need to handle the case of payload being produced via blinded flow.

So I think we need to expose the winning value in the `HeaderWithFallbackData` created in `ExecutionBuilderModule::builderGetHeader`.

I'd rename the field to `executionPayloadValueFuture` instead of `blockValue`

Reply:
Done and updated the tests accordingly